### PR TITLE
Update dependencies and translations to the most recent versions

### PR DESCRIPTION
**Pull request by timvandermeij**
The commit messages contain more details about the individual changes. This is most likely the final update before the next release.

### CODE REVIEW

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you.

File: gulpfile.mjs [hunk 1]
```diff
@@ -2250,7 +2250,7 @@ function packageJson() {
     bugs: DIST_BUGS_URL,
     license: DIST_LICENSE,
     optionalDependencies: {
-      "@napi-rs/canvas": "^0.1.62",
+      "@napi-rs/canvas": "^0.1.63",
```
Comment (Brooooooklyn) [inline gulpfile.mjs:2253]: Please upgrade to `0.1.64`, as `0.1.63` contains a regression that may cause an abnormal exit when using text operations on Windows: https://github.com/Brooooooklyn/canvas/pull/953, `0.1.62` does not have this issue, and it was accidentally introduced in `0.1.63` when adding support for the `riscv` platform.

Comment (timvandermeij) [reply gulpfile.mjs]: Thanks for the heads-up; done in the new commit.

**Pull request merged**